problems/prob1.rb

```diff
@@ -0,0 +1,10 @@
+#Given an integer array of size N, compute
+#the sum of all even numbers in this array
+
+def sum_of_even_numbers(numbers_array)
+  if numbers_array.empty?
```

Comment:
This is nitpicking, but after you will write some more ruby I bet you will agree.

Open your favourite terminal and type `irb`, you should get a nice prompt where you can type ruby code and press enter to run it, type there `[].sum` and notice what happens, come back to this problem and take a second look at it ;)

Reply:
I refactored the problem in the latest commit.